crates/testing/tests/standalone_integration_test.rs

```diff
@@ -139,3 +139,27 @@ fn test_call_query_macro() {
         },
     );
 }
+
+#[test]
+#[serial]
+fn test_index_scans() {
```

Comment:
```suggestion
/// Assert that all of the timing spans in the `perf-test` module's `test_index_scans` reducer
/// are measured in micro- or nanoseconds.
///
/// This timing is a proxy for use of a multi-column index,
/// under the assumption that, because indexes provide an algorithmic speedup,
/// failing to use the index will result in execution being orders of magnitude worse,
// and taking milliseconds.
fn test_index_scans() {
```